_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -1426,58 +1459,149 @@ Set the variables for the `account-baseline-app` module in this environment in t
+  aws_region     = "us-east-1"
+  aws_account_id = local.accounts["logs"]
 
   # Prefix all resources with this name
-  name_prefix    = "<COMPANY_NAME>-logs"
+  name_prefix = "acme-logs"
+ ################################
+  # Parameters for AWS Config
+  ################################
+  # Send Config logs to the common S3 bucket.
+  config_s3_bucket_name = local.config_s3_bucket_name
 
-  # Set to true to create an sns topic in this account for sending AWS Config notifications (e.g., if this is the logs account).
-  config_should_create_sns_topic = true
+  # Send Config logs and events to the logs account.
+  config_central_account_id = local.accounts.logs
 
-  # Use the S3 bucket that was already created in this logs account by account-baseline-root
-  config_s3_bucket_name = "<CONFIG_BUCKET_NAME>"
+  # Do not allow objects in the Config S3 bucket to be forcefully removed during destroy operations.
+  config_force_destroy = false
 
-  # Set this to false because this is the account to aggregate Config data
-  config_aggregate_config_data_in_external_account = false
-  config_central_account_id                        = null
+  #  This is the Logs account, so we create the SNS topic for aggregating Config logs from all accounts.
+  config_should_create_sns_topic = true
 
-  # The bucket was created by account-baseline-root in this account
-  config_should_create_s3_bucket = false
+  # All of the other accounts send logs to this account.
+  config_linked_accounts = [
+  for name, id in local.accounts :
+  id if name != "logs"
+  ]
 
-  # The name of the SNS topic in where AWS Config notifications will be sent. Can be in the same account or in another account.
-  config_sns_topic_name = "<DEFINE_SNS_TOPIC_NAME>"
+  ################################
+  # Parameters for CloudTrail
+  ################################
 
-  # Specify all of the account IDs that should be able to write to the AWS Config in the `logs` account. You might want to set this to the `security` child account, or `dev`, `prod` or `stage` AWS Account IDs.
-  config_linked_accounts = ["<ANY_CHILD_AWS_ACCOUNT_ID>", "<SECURITY_ACCOUNT_ID>", "<DEV_ACCOUNT_ID>"]
+  # Send CloudTrail logs to the common S3 bucket.
+  cloudtrail_s3_bucket_name = local.cloudtrail_s3_bucket_name
 
-  # Create a KMS master key that can be used to encrypt CloudTrail data. Make the logs account the admin/user of this key.
-  # In practice, this means you can use IAM policies in the logs account to specify who has access to this key.
-  cloudtrail_kms_key_administrator_iam_arns = ["arn:aws:iam::<LOGS_ACCOUNT_ID>:root"]
-  cloudtrail_kms_key_user_iam_arns          = ["arn:aws:iam::<LOGS_ACCOUNT_ID>:root"]
+  # All of the other accounts send logs to this account.
+  cloudtrail_allow_kms_describe_key_to_external_aws_accounts = true
+  cloudtrail_external_aws_account_ids_with_write_access = [
+  for name, id in local.accounts :
+  id if name != "logs"
```

Comment:
NIT: In my view, this is a line under the for loop, so it seems like it should be further indented. (We need to update the ref arch generation to reflect this too.)